README.md

```diff
@@ -135,12 +158,13 @@ The methods work with auto it variable
 ```nim
 sequence --> map(op)
 ```
-Map each item in the list to a new value.
+Map each item in the list to a new value. 
```

Comment:
Map each item in the array to a new value

Reply:
... rewrote everything with "collection" instead of list or sequence